pkg/pusher/pusher.go

```diff
@@ -202,9 +202,14 @@ func (s *Service) chunksWorker(warmupTime time.Duration, tracer *tracing.Tracer)
 				}
 			case apiC := <-s.smuggler:
 				go func() {
-					for op := range apiC {
```

Comment:
this is fine, we don't need to touch this one.

Reply:
just double checked it and it looks like change from PR is needed.

tests fail with this error without it:
```
goleak: Errors on successful test run: found unexpected goroutines:
[Goroutine 13 in state chan receive, with github.com/ethersphere/bee/pkg/pusher.(*Service).chunksWorker.func5.1 on top of the stack:
goroutine 13 [chan receive]:
github.com/ethersphere/bee/pkg/pusher.(*Service).chunksWorker.func5.1()
        /home/vlado/Workspace/bee/pkg/pusher/pusher.go:205 +0x5b
created by github.com/ethersphere/bee/pkg/pusher.(*Service).chunksWorker.func5
        /home/vlado/Workspace/bee/pkg/pusher/pusher.go:204 +0x28a
```